decentraland/unity-renderer
# Fix: gray screen flash on nav map

## What does this PR change?
Fixes https://github.com/decentraland/unity-renderer/issues/2071

- Add a small controller to disable/enable white background when loading NavMap
- Refactored ExploreV2Controller/View to reduce duplication of the code
- Move responsibility of changing Model from ExploreV2View to ExploreV2Controller

## How to test the changes?
Test 1
1. Open NavMap via shortcut and observe that it loads without the white splash
2. Switch between other tabs (Explore, Backpack, Settings...) and NavMap via UI or Shorcuts and observe no white splash when opening/closing NavMap

Test 2
1. Open NavMap
2. Close ExploreUI via Close button on the right hand side
3. Open Settings via shortcut (P)
4. Observe that Settings are opening on the white background (and you don't see 3d world behind it)

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
